### ISSUE TITLE
Improve Jetpack stats tabs

### ISSUE THREAD

**Issue by fditrapani** [Automattic/wp-calypso#100507]
## Problem

The tabs on the main Jetpack Stats card don't look like tabs. It's not immediately clear that they're interactive:

<img width="1839" alt="Image" src="https://github.com/user-attachments/assets/beb21bad-db51-4d35-98e7-56464f8b5940" />

## Where
Visit the Jetpack Stats on any site. Preferably one with data.

**Comment by oandregal** [comment]
@fditrapani are you available to provide design support for this issue, or should I engage other designers? If so, who should I ping? I ask because I see you've created hundreds of issues, and I suppose you won't have the bandwidth to care for all of them.

I'm sharing some thoughts, but I'd welcome design feedback on the way forward.

## Rest state

We should aim to use the same patterns across the sections. Just by comparing to what similar sections look like, I see a few things we can improve here:

- Typography
- Background of the section 

Does this align with the design expectations for this section?

| Page | Header |
| --- | --- |
| Stats | <img width="2557" alt="Image" src="https://github.com/user-attachments/assets/23216c93-7b5b-4c11-ad54-b9bcbb2e8439" /> |
| Protect | <img width="2557" alt="Image" src="https://github.com/user-attachments/assets/665ee8ca-a99d-43e2-9d96-8c835d09d30d" /> |
| Settings | <img width="2557" alt="Image" src="https://github.com/user-attachments/assets/bb07942a-2c81-4b48-9a95-a5d3b979185d" /> |
| Dashboard | <img width="2557" alt="Image" src="https://github.com/user-attachments/assets/f4ee511c-b557-4643-b189-828529595db5" /> |

## Hover state

Different patterns in use, this needs consolidation across sections. Which pattern do we go with? I see two:

1. Green on hover.
2. Display an underline under the hovered section

https://github.com/user-attachments/assets/bfaab4e0-ee0d-4a9d-8301-326364069ed1

https://github.com/user-attachments/assets/bddceee0-93e8-4dd7-9edf-ef4bad7b5ced

https://github.com/user-attachments/assets/d39b7112-445d-488e-8c6a-5a142cbdd870

https://github.com/user-attachments/assets/a1ee1567-d322-4290-8e0b-7f6142655170

## Focus state

Different patterns in use, it needs consolidation across section. Which pattern do we go with? I see two:

1. Green
2. Box. But, in all the examples, the top side is hidden – it shouldn't.

| Page | Header |
| --- | --- |
| Stats | <img width="1279" alt="Image" src="https://github.com/user-attachments/assets/dca2f78f-f238-4dc4-8c99-c53c3dc4c131" /> |
| Protect | <img width="1279" alt="Image" src="https://github.com/user-attachments/assets/396089e8-fae2-461d-b61d-727a0dc64ecf" /> |
| Settings | <img width="1279" alt="Image" src="https://github.com/user-attachments/assets/2e918e55-7b64-4494-95ac-2ea90798ccdd" /> |
| Dashboard | <img width="1279" alt="Image" src="https://github.com/user-attachments/assets/6a7a024a-19db-4cad-af75-5898941a018f" /> |

**Comment by ederrengifo** [comment]
It seems this issue lacks of important context to consider:
- We cannot apply consistent patterns for tabs in this case because these are not tabs used for navigation but rather an element part of the chart, which is a more unique scenario.
- We are currently exploring consistency for charts and cards in coordination with Woo Analytics, so there will be improvements in the near future for the whole chart experience, it seems it would be a waste of efforts to make an update now. Making tabs look more interactive is definitely a good feedback for that update though.
- Unless we overlooked, there is no issues reported by users about this being a problem for the UX. This element has been already out for more than 2 years since the latest redesign and it's already based in an old behavior. In the past, we received backlash when we introduced changes that users perceived as unrequested, so unless we make an upgrade to the whole chart experience that can help it to be more valuable, if we focus only on this change, we are turning it into a high risk / low reward type of update and I don't think it's worth prioritizing.

cc @oandregal @kangzj @crisbusquets

**Comment by crisbusquets** [comment]
Thanks for chiming in, @ederrengifo 🙏  I think we can close this one.

> We are currently exploring consistency for charts and cards in coordination with Woo Analytics, so there will be improvements in the near future for the whole chart experience, it seems it would be a waste of efforts to make an update now. Making tabs look more interactive is definitely a good feedback for that update though.

Good to know! The more consistency, the better 🚀

The visual inconsistencies that @oandregal shared [above](https://github.com/Automattic/wp-calypso/issues/100507#issuecomment-2718793423) seem to be a "Code Blue" (or green?) issue for Jetpack.
